src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -31,6 +31,14 @@ public TemplateCache(ScanResult?[] scanResults, Dictionary<string, DateTime> mou
                 }
                 foreach (ITemplate template in scanResult.Templates)
                 {
+                    if (templateDeduplicationDictionary.ContainsKey(template.Identity))
+                    {
+                        _logger.LogWarning(string.Format(
+                            LocalizableStrings.TemplatePackageManager_Warning_DetectedTemplatesIdentityConflict,
+                            template.Name,
+                            templateDeduplicationDictionary[template.Identity].Template.Name,
+                            templateDeduplicationDictionary[template.Identity].Template.MountPointUri));
+                    }
```

Comment:
is it possible to get package display name instead? Mount Point URI is a path.

Reply:
What would be the final message example in case 2 template packages (NuGet based) have same template?